libraries/lts/include/mcrl2/lts/detail/liblts_bisim_m.h

```diff
@@ -0,0 +1,579 @@
+// Author(s): Jan Martens
+//
+// Copyright: see the accompanying file COPYING or copy at
+// https://github.com/mCRL2org/mCRL2/blob/master/COPYING
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+/// \file lts/detail/liblts_bisim_m.h
+///
+/// \brief Partition refinement algorithm for guaruanteed minimal depth
+/// counter-examples.
+///
+/// \details 
+#ifndef _LIBLTS_BISIM_M
+#define _LIBLTS_BISIM_M
+
+#include <fstream>
+#include "mcrl2/modal_formula/state_formula.h"
+#include "mcrl2/lts/lts_utilities.h"
+#include "mcrl2/lts/detail/liblts_scc.h"
+#include "mcrl2/lts/detail/liblts_merge.h"
+#include "mcrl2/lts/lts_aut.h"
+#include "mcrl2/lts/lts_fsm.h"
+#include "mcrl2/lts/lts_dot.h"
+
+namespace mcrl2
+{
+namespace lts
+{
+namespace detail
+{
+template < class LTS_TYPE>
+class bisim_partitioner_counter_example {
+public:
+    /** \brief Creates a bisimulation partitioner for an LTS.
+    *  \details TODO.
```

Comment:
I would suggest to either provide details, not sure if \warning is actually a valid doxygen directive so maybe that should be under details.